subsys/bluetooth/host/audio/mcc.c

```diff
@@ -1785,6 +1882,37 @@ int bt_mcc_read_next_track_obj_id(struct bt_conn *conn)
 	return err;
 }
 
+int bt_mcc_set_next_track_obj_id(struct bt_conn *conn, uint64_t obj_id)
```

Comment:
```suggestion
int bt_mcc_next_track_obj_id_set(struct bt_conn *conn, uint64_t obj_id)
```

Reply:
Should we add a check for the `obj_id` value? It shall be greater than or equal to `0x100` or something, right? (applies multiple places)

Reply:
Added it to the three added functions.
Made upstream PR to get a #define into OTS